src/components/projects.tsx

```
@@ -0,0 +1,118 @@
+  All: "text-blue-400",
+  Education: "text-purple-400",
+  Environment: "text-green-400",
+  Technology: "text-cyan-400",
+  "Arts & Culture": "text-pink-400",
+  Wellness: "text-red-400",
+  Community: "text-yellow-400",
+}
+
+export default function ProjectPage({ params }: { params: { id: string } }) {
+  const [isDonationOverlayOpen, setIsDonationOverlayOpen] = useState(false)
+  const project = projectsData.find((p) => p.id === Number.parseInt(params.id))
```

ESLint

[error] Missing semicolon.

(style/semi)